packages/terra-alert/src/terra-dev-site/doc/example/LongTextExample.jsx

```diff
@@ -12,6 +12,7 @@ const LongTextExample = () => {
         Four score and seven years ago our fathers brought forth on this continent, a new nation, conceived in Liberty, and dedicated to the proposition that all men are created equal.
         <Button
           variant="de-emphasis"
+          isCompact
```

Comment:
Rather than trying to fix Button, which will never have proper alignment inline with text, might be easiest to  resolve by switching out the Toggle in the example for the `terra-show-hide` component, even though it adds a new line.

Reply:
Backup option could be we swap out the toggle to use a hyperlink (which would stay inline) which launches a basic javascript alert with the hidden/full text.

Reply:
I replaced toggle with show/hide on the Heroku deployment.